bundles/pascal/misc/example.pas

```diff
@@ -0,0 +1,53 @@
+{ Comments }
```

Comment:
@nilnor Well, that's odd...

Here's a minimal test case:

```pascal
{




















}
```

Basically, it has to be a multiline comment over 20 lines long. I'm not quite sure what the problem is; it also occurs with any other forms of multi-line comment.

Looking into it, thought this may take a while...